tools/test-js-with-node

```diff
@@ -231,7 +231,6 @@ EXEMPT_FILES = make_set(
         "web/src/zulip.js",
         "web/src/zulip_test.js",
         "web/tests/lib/mdiff.js",
-        "web/tests/lib/zjquery_element.js",
```

Comment:
What's the story with this change? It seems surprising this would be related.

Reply:
What's happening here is adding `$("#compose-textarea").toggleClass("invalid", false);` to `compose_actions.js` actually increases the line coverage of `web/tests/lib/zjquery_element.js` to almost 95%, hence had to remove it from the exclusion list.

To be precise the line coverage of the `else` block of the following block of code got increased, which makes sense as well:
```
toggleClass(class_name, add) {
            if (add) {
                classes.set(class_name, true);
            } else {
                classes.delete(class_name);
            }
            return $self;
        },
```